packages/block-editor/src/components/block-list/style.scss

```diff
@@ -495,3 +505,7 @@
 		opacity: 0;
 	}
 }
+
+.is-dragging-components-draggable .components-tooltip {
+	display: none;
```

Comment:
@youknowriad, I added this as the tooltip appears shortly after you start dragging, and it just floats in space. However I feel this may be the wrong fix.

Additionally, it feels as if dragging a single slot doesn't work, it only works if dragging more than two blocks worth of distance. I suspect it's related to the tooltip appearing after starting the dragging operation but I'm not sure. 

If you have time, I would appreciate your help with this one.